applications/CompressiblePotentialFlowApplication/tests/potential_flow_test_factory.py

```diff
@@ -36,7 +36,7 @@ def setUp(self):
         # Set to true to get post-process files for the test
         self.print_output = False
 
-    @UnitTest.skipIfApplicationsNotAvailable("ExternalSolversApplication", "HDF5Application")
+    @UnitTest.skipIfApplicationsNotAvailable("HDF5Application")
```

Comment:
isn't is now depending on the linearsolversapp?

Reply:
True :sweat_smile: